code/modules/mob/living/simple_animal/abnormality/teth/pale_horse.dm

```diff
@@ -0,0 +1,221 @@
+	var/turf/target_turf = get_turf(src)
+	for(var/turf/T in view(1, target_turf))
+		if(prob(70))
+			continue
+		new /obj/effect/palefog(T)
+
+/mob/living/simple_animal/hostile/abnormality/pale_horse/Move() //more damaging fog when moving
+	var/turf/target_turf = get_turf(src)
+	for(var/turf/T in view(0, target_turf))
+		new /obj/effect/palefog(T)
+	..()
+
+/mob/living/simple_animal/hostile/abnormality/pale_horse/AttackingTarget()
+	if(!isliving(target))
+		return FALSE
+
+	var/mob/living/L = target
+	if((L.health > 0 || L.stat == DEAD))
+		var/datum/status_effect/mortis/M = L.has_status_effect(/datum/status_effect/mortis)
+		if(!M)
+			say("#Neigh.")
+			L.apply_status_effect(STATUS_EFFECT_MORTIS)
+			playsound(L, 'sound/abnormalities/palehorse/debuff.ogg', 50, 0, -1)
+		LoseTarget(L)
+		return FALSE
+	. = ..()
+
+	if(ishuman(target)) //blow up corpses to cause bad things to happen
+		var/mob/living/carbon/human/T = target
+		addtimer(CALLBACK(src, .proc/TryTeleport), 5)
+		playsound(get_turf(src), 'sound/abnormalities/palehorse/kill.ogg', 50, 0, 8)
+		visible_message("<span class='danger'>[T] is reduced to ashes!</span>")
+		for(var/mob/living/simple_animal/hostile/abnormality/P in range(pulse_range, src))
+			if(!(P.IsContained()))
+				continue
+			P.datum_reference.qliphoth_change(-1)
+		for(var/turf/F in view(pulse_range, src))
+			new /obj/effect/palefog(F)
+			for(var/mob/living/H in F)
+				if(faction_check_mob(H))
+					continue
+				if(H.health < 0 && ishuman(H))
+					H.dust()
+		T.dust()
+
+//MOSB corpse-seeking behavior
+/mob/living/simple_animal/hostile/abnormality/pale_horse/GiveTarget(new_target)
+	if(LAZYLEN(patrol_path))
+		if(locate(/mob/living/carbon/human) in patrol_path[patrol_path.len])
+			return // IGNORE EVERYTHING, CHARGE!!!!
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/pale_horse/PickTarget(list/Targets) //Attack corpses first if there are any
+	var/list/highest_priority = list()
+	var/list/lower_priority = list()
+	for(var/mob/living/L in Targets)
+		if(!CanAttack(L))
+			continue
+		if(L.health < 0 || L.stat == DEAD)
+			if(ishuman(L))
+				highest_priority += L
+			else
+				lower_priority += L
+		else if(L.health < L.maxHealth*0.5)
+			lower_priority += L
+	if(LAZYLEN(highest_priority))
+		return pick(highest_priority)
+	if(LAZYLEN(lower_priority))
+		return pick(lower_priority)
+	return ..()
+
+//objects
+/obj/effect/palefog
+	name = "fog"
+	desc = "A vapor that faintly smells of alcohol"
+	icon_state = "smoke"
+	move_force = INFINITY
+	pull_force = INFINITY
+	generic_canpass = FALSE
+	movement_type = PHASING | FLYING
+	var/mine_damage = 5 // 5 pale damage
+	var/list/faction = list("hostile")
+	layer = POINT_LAYER	//We want this HIGH. SUPER HIGH. We want it so that you can absolutely, guaranteed, see exactly what is about to hit you.
+
+/obj/effect/palefog/Initialize()
+	..()
+	addtimer(CALLBACK(src, .proc/explode), 0.5 SECONDS)
```

Comment:
This is pretty fast, all things considered.

Reply:
It's meant to be. Each explosion only deals 5 pale damage, and has tiny range.

Reply:
Yes but the range is 1 tile around them, and they can spawn on, or directly around, the horse. That's "only" 5 pale from potentially multiple sources at once on top of the ability to be double due to the debuff. That's "only" 10% of someone's health from a single cloud that could've spawned _on_ them.

Reply:
Changed how this works, but it is still subject to experimentation to see how balance factors in.